File: CourseHomeworkEleven/app/src/main/java/com/example/coursehomeworknine/WindObject.java
package com.example.coursehomeworknine;

import com.google.gson.annotations.SerializedName;

public class WindObject {

    @SerializedName("speed")
    private String speed;

    public String getSpeed() {
        return speed;
    }

    public void setSpeed(String speed) {
        this.speed = speed;
    }
}

File: CourseHomeworkTen/app/src/main/java/com/example/coursehomeworkten/WindObject.kt
package com.example.coursehomeworkten

import com.google.gson.annotations.SerializedName




class WindObject{
    @SerializedName("speed")
    private var speed: String? = null

    fun getSpeed(): String? {
        return speed
    }

    fun setSpeed(speed: String?) {
        this.speed = speed
    }
}